homeassistant/components/israel_rail/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""The Israel Rail component."""
+
+import logging
+
+from israelrailapi import TrainSchedule
+
+from homeassistant import config_entries, core
+from homeassistant.const import Platform
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_DESTINATION, CONF_START, DOMAIN
+from .coordinator import IsraelRailDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+PLATFORMS: list[Platform] = [Platform.SENSOR]
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
```

Comment:
```suggestion
    hass: HomeAssistant, entry: ConfigEntry
```
I think we can use entry.runtime data here

Reply:
cool
Done